test/rendering/__fixtures__/scenario_3.fix.js

```diff
@@ -0,0 +1,104 @@
+export default () => ({
+  type: 'sn-action-button',
+  genericObjects: [
+    {
+      getLayout: {
+        qInfo: {
+          qId: 'HNpPeCV',
+          qType: 'sn-action-button',
+        },
+        qMeta: {
+          privileges: ['read', 'update', 'delete', 'exportdata'],
+        },
+        qSelectionInfo: {},
+        actions: [],
+        navigation: {
+          action: 'none',
+          sheet: '',
+          story: '',
+          websiteUrl: '',
+          sameWindow: false,
+        },
+        useEnabledCondition: false,
+        enabledCondition: 1,
+        showTitles: false,
+        title: '',
+        subtitle: '',
+        footnote: '',
+        disableNavMenu: false,
+        showDetails: false,
+        style: {
+          label: 'Button',
+          font: {
+            fontFamily: 'Source Sans Pro',
+            sizeBehavior: 'relative',
+            size: 0.46,
+            useColorExpression: false,
+            color: {
+              index: 8,
+              color: '#8a85c6',
+            },
+            colorExpression: '',
+            style: {
+              bold: true,
+              italic: true,
+              underline: false,
+            },
+            align: 'right',
+          },
+          background: {
+            useColorExpression: false,
+            color: {
+              index: 2,
+              color: '#99cfcd',
+            },
+            colorExpression: '',
+            mode: 'media',
+            size: 'fitWidth',
+            position: 'top-center',
+            url: {
+              qStaticContentUrl: {
+                qUrl: '/content/default/qlik-multi-kpi-update/multikpi.png',
+              },
+            },
+          },
```

Comment:
@niekvanstaveren I have some background image settings here but I don't see any sign of the it there on the button! ... either on the scenario_1 testcase which test useImage.

Reply:
Pretty sure this image is not available in this rendering test. Looks like this was copied over from the multi kpi. We should not have these options here if they are not used for the tests